src/shared/components/ViewForm/SparqlQueryResults.tsx

```diff
@@ -23,6 +24,34 @@ export type Entry = {
   type: string;
 };
 
+const getUrl = (entry: string) => {
```

Comment:
we could probably move the function to utils file and test it?

Reply:
I have move the bulk of logic to utils/added tests for it.